resolvers.js

```diff
@@ -79,9 +82,19 @@ const resolvers = {
         // TODO: Double check defaults
         title: 'untitled',
         currentView: 'table',
-        notes: []
+        notes: [],
+        categories: []
+      }).save();
+
+      const newCategory = await new Category({
+        name: 'Non-categorised',
+        notes: [],
+        databaseId: newDatabase._id
       }).save();
 
+      newDatabase.categories.push(newCategory._id);
+      await newDatabase.save();
+
       userDocument.databases.push(newDatabase._id);
       await userDocument.save();
```

Comment:
shld use the mongoose default field for this instead of inserting 'Non-categorised' after the document has been created